network-store-model/src/main/java/com/powsybl/network/store/model/LineAttributes.java

```diff
@@ -124,4 +125,79 @@ public class LineAttributes extends AbstractAttributes implements BranchAttribut
 
     @Schema(description = "Branch status")
     private String branchStatus;
+
+    @Override
+    @JsonIgnore
+    public LimitsAttributes getLimits(TemporaryLimitType type, int side) {
+        switch (type) {
+            case CURRENT_LIMIT:
+                if (side == 1) {
+                    return currentLimits1;
+                }
+                if (side == 2) {
+                    return currentLimits2;
+                }
+                throw new IllegalArgumentException("Unknown side for line");
```

Comment:
same move this check at the beginning of the function to factorize some code

Reply:
Done